test/testcases/block/06_codeblock/with_lang_in_fenced_block_name_with_dash.text

```diff
@@ -0,0 +1,33 @@
+~~~ ruby-trunk
+def what?
+  42
+end
+~~~
+
```

Comment:
To make sure, that a language with a dash behaves the same like a language without a dash